### PR TITLE
Add snippet primes

### PR DESCRIPTION
**Pull request by kriadmin**
## Description

Add a snippet primes to generate prime numbers

## What does your PR belong to?
- [ ] Website
- [x] Snippets
- [ ] General / Things regarding the repository (like CI Integration)

## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Enhancement (non-breaking improvement of a snippet)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have checked that the changes are working properly
- [x] I have checked that there isn't any PR doing the same
- [x] I have read the **CONTRIBUTING** document.

### CODE REVIEW

File: snippets/primes.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+### primes 
+
+It generates primes till a given number. 
+
+It works with the Sieve of Eratosthenes
```
Comment (iamsoorena) [inline snippets/primes.md]: please explain about interbal workings of your function. how does it work? which built in functions you use to achieve your goal?

File: snippets/primes.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+```js
+const primes = num =>{
```
Comment (iamsoorena) [inline snippets/primes.md]: put a space after `=>`

**Comment by kriadmin** [comment]
@iamsoorena Updated

**Comment by iamsoorena** [comment]
@kriadmin good job. thanks for your contribution. hope to see more from you.

**Pull request merged**